rocAL/rocAL/include/api/rocal_api_data_loaders.h

```diff
@@ -747,5 +747,61 @@ extern "C"  RocalImage  ROCAL_API_CALL rocalJpegCaffe2LMDBRecordSourcePartialSin
+/// \param shuffle Determines if the user wants to shuffle the dataset or not.
+/// \param loop Determines if the user wants to indefinitely loops through images or not.
+/// \param downmix Determines weather the decoded audio is downmixed or not.
+/// \param max_frames The maximum frames of the decoded audio.
+/// \param max_channels The maximum channels of the decoded audio.
+/// \return Reference to the output image
+extern "C"  RocalTensor  ROCAL_API_CALL rocalAudioFileSource(RocalContext context,
+                                                            const char* source_path,
+                                                            unsigned internal_shard_count,
+                                                            bool is_output,
+                                                            bool shuffle = false,
+                                                            bool loop = false,
+                                                            float sample_rate = 0.0,
+                                                            bool downmix = false,
+                                                            unsigned max_frames = 1,
+                                                            unsigned max_channels = 1);
+
+/// Creates Audio file reader and decoder. It allocates the resources and objects required to read and decode audio files stored on the file systems. It has internal sharding capability to load/decode in parallel is user wants.
+/// If the files are not in standard audio compression formats they will be ignored.
+/// \param context Rocal context
+/// \param source_path A NULL terminated char string pointing to the location on the disk
+/// \param source_file_list_path A NULL terminated char string pointing to the file list location on the disk
+/// \param shard_id Shard id for this loader
+/// \param shard_count Defines the parallelism level by internally sharding the input dataset and load/decode using multiple decoder/loader instances. Using shard counts bigger than 1 improves the load/decode performance if compute resources (CPU cores) are available.
+/// \param is_output Determines if the user wants the loaded images to be part of the output or not.
+/// \param shuffle Determines if the user wants to shuffle the dataset or not.
+/// \param loop Determines if the user wants to indefinitely loops through images or not.
+/// \param downmix Determines weather the decoded audio is downmixed or not.
+/// \param max_frames The maximum frames of the decoded audio.
+/// \param max_channels The maximum channels of the decoded audio.
+/// \param storage_type Determines the storage type
+/// \param stick_to_shard Determines weather or not the dataset when sharded should stick to a single shards dataset or considered in a round robin fashion.
+/// \param shard_size Provides the size of the shard for an iterator 
+/// \return Reference to the output image
```

Comment:
Instead of saying reference to output image, you mention reference to output tensor

Reply:
Changed it.